jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibPlugin.java

```diff
@@ -32,23 +32,29 @@ public void apply(Project project) {
     checkGradleVersion();
 
     JibExtension jibExtension = project.getExtensions().create("jib", JibExtension.class, project);
-    Task classesTask = project.getTasks().getByPath("classes");
 
-    project
-        .getTasks()
-        .create("jib", BuildImageTask.class)
-        .setJibExtension(jibExtension)
-        .dependsOn(classesTask);
-    project
-        .getTasks()
-        .create("jibDockerContext", DockerContextTask.class)
-        .setJibExtension(jibExtension)
-        .dependsOn(classesTask);
-    project
-        .getTasks()
-        .create("jibBuildDocker", BuildDockerTask.class)
-        .setJibExtension(jibExtension)
-        .dependsOn(classesTask);
+    Task buildImageTask =
+        project.getTasks().create("jib", BuildImageTask.class).setJibExtension(jibExtension);
+    Task dockerContextTask =
+        project
+            .getTasks()
+            .create("jibDockerContext", DockerContextTask.class)
+            .setJibExtension(jibExtension);
+    Task buildDockerTask =
+        project
+            .getTasks()
+            .create("jibBuildDocker", BuildDockerTask.class)
+            .setJibExtension(jibExtension);
+
+    // Has all tasks depend on the 'classes' task.
+    project.afterEvaluate(
+        ignored -> {
```

Comment:
I wonder if the project object in the afterEvaluate block has different properties than the one calling afterEvaluate?

Reply:
the one we've `ignored`

Reply:
Hmm, okay I'll change to use the one that was ignored.